pgmoon/init.lua

```diff
@@ -7,7 +7,7 @@ do
   rshift, lshift, band = _obj_0.rshift, _obj_0.lshift, _obj_0.band
 end
 local unpack = table.unpack or unpack
-local VERSION = "1.11.0"
+local VERSION = "1.11.1"
```

Comment:
based on #2 it looks like `pgmoon-mashape` bumped a full major (`2.0.0`) so I think we will want to do something similar for this release as well. we might need to bring this up in standup.